src/registrar/forms/domain_request_wizard.py

```diff
@@ -736,7 +736,14 @@ class NoOtherContactsForm(BaseDeletableRegistrarForm):
         required=True,
         # label has to end in a space to get the label_suffix to show
         label=("No other employees rationale"),
-        widget=forms.Textarea(),
+        widget=forms.Textarea(
+            attrs={
+                "aria-label": "You don’t need to provide names of other employees now, \
```

Comment:
#2918 and #2915

Reply:
@erinysong I don't see the "No other employees" text in the ANDI Output mentioned in #2918 
<img width="1319" alt="Screenshot 2024-12-19 at 9 31 38 AM" src="https://github.com/user-attachments/assets/31fd4d34-14ab-4961-b158-7296a0ff0b00" />

Do I need to focus on the text area differently?